app/move/fields/update.js

```diff
@@ -0,0 +1,23 @@
+const { cloneDeep } = require('lodash')
+const createFields = require('./create')
+
+const updateFields = cloneDeep(createFields)
+
+updateFields.police_national_computer = {
+  ...updateFields.police_national_computer,
+  updateProtect: true,
```

Comment:
What about `readOnly`? Feels just as clear and a more common term for this. It also matches how we use this term in the API.

Reply:
Done